<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_window</class>
 <widget class="QMainWindow" name="student_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>1121</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>找书</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>811</width>
        <height>271</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Book id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book name</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>writer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>press</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>storage</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>120</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Al Tarikh</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>find books</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>61</y>
        <width>232</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>book id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>book name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>writer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>press</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="find_bookid"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_bookname"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_writer"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_press"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>还书</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>book id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="return_bookid"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="ret">
         <property name="text">
          <string>return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>10</y>
        <width>781</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Al Tarikh</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>already borrowed and kept</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fresh_borrow">
           <property name="text">
            <string>更新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>197</number>
         </attribute>
         <column>
          <property name="text">
           <string>Book id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Book name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>borrow time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>return time</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>351</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Al Tarikh</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Borrow books</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Al Tarikh</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>book id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="borrow_bookid"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="borrow">
         <property name="text">
          <string>submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>351</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>student id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>student name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLineEdit" name="read_id">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="read_name">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>log out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
